README.md

```diff
@@ -263,6 +263,28 @@ API clients (Web UIs, Mobile, etc.)
 * https://github.com/openMF has more related proejcts
 
 
+Generate Apache Fineract API Client
+============
+
+Apache Fineract supports client code generation using [Swagger Codegen](https://github.com/swagger-api/swagger-codegen). Project supports [all clients](https://github.com/swagger-api/swagger-codegen#overview) supported by Swagger Codegen. It uses [OpenAPI Specification Version 3.0.3](https://swagger.io/specification/).
+
+To generate code, execute the following commands:
+
+- **./gradlew build**
+- **./gradlew generateSwaggerCode**
+
+The client code will be generated in the directory **build/swagger-code-fineract**
+
+To build the generated code:
+
+- Navigate to **build/swagger-code-fineract** directory
+- **./gradlew build**
+
+To customize the generated code:
+
+- Open **fineract-provider/config/swagger/config.json**
+- Refer to [this](https://gist.github.com/Grandolf49/2d222c3d1d0b834ad8f02b5fde8b4c14) for available options in config.json
```

Comment:
If you move it to a separate MD docs file, you could just "inline" your entire Gist? Better. Actually, perhaps even better you could link to the appropriate docs page of Swagger CodeGen where they document this? Assuming they have proper doc! :imp: That way, it stays up to date, as we'll upgrade the plugin in the future.

Reply:
I spent a lot of time trying to find where they have documented these options. 😅 [This](https://github.com/swagger-api/swagger-codegen/issues/7795) thread was useful but just a workaround.

I also came across [this](https://openapi-generator.tech/docs/generators/java) which seems like the official documentation from the OpenAPI Team. 

I think it's just better to move this to a separate doc. I'll put a link to this in the doc as you suggested

Reply:
@vorburger I have done the necessary changes and created a new client.md. I could also spot some minor corrections in the main readme.md file. So I have updated those too! I hope that's okay! 😅

Let me know if I need to change anything else.